src/hotspot/os/linux/gc/z/zMountPoint_linux.cpp

```diff
@@ -61,11 +61,11 @@ char* ZMountPoint::get_mountpoint(const char* line, const char* filesystem) cons
       strcmp(line_filesystem, filesystem) != 0 ||
       access(line_mountpoint, R_OK|W_OK|X_OK) != 0) {
     // Not a matching or accessible filesystem
-    os::free(line_mountpoint);
+    ALLOW_C_FUNCTION(::free, ::free(line_mountpoint);) // *not* os::free
```

Comment:
Can you please extend the comment like this: "sscanf, using %m, will return malloce'd memory. Needs raw ::free, not os::free."